modules/ensemble/lib/widget/confirmation_input.dart

```diff
@@ -35,7 +37,7 @@ class ConfirmationInput extends StatefulWidget
           _controller.fieldType = Utils.optionalString(input),
       'inputType': (type) => _controller.inputType = Utils.optionalString(type),
       'obscureText': (type) => _controller.obscureText = Utils.optionalString(type),
```

Comment:
@TheNoumanDev `obscureText` should be a boolean. We need to be consistent with TextInput. See Vu's message on discord as well. We need two things - 

obscureText: boolean 
obscureSymbol: string | icon

if obscureText  == true, only then you process the obscureSymbol. And the default obscureSymbol is * Please fix yoru code and repost

Reply:
Hey @Khurram!
Below is just description of wot happening bcz first i kept it boolean and sharjeel bhai told me to not change types that are already defined😅 , then bcz of Icon requirement i changed neccesary property and code but kept rest same as @vusters approved last changes but i understand it should be boolean to keep it consistent, should i change it or keep it same? 🙂 

It actually has 3 states defined in enum that's why it is in string:
`none`: to display plain text (default)
`password`: to have obscure text just as `*`.
`custom`: to have obscure text but with their own choice of character or Icon and `obscureSymbol` will only be processed if `custom` is selected.

Reply:
we can make it backward compatible as follows - 

1. if you get `null` returned by `Utils.optionalBool(obscure)`, parse it as a string
2. update the schema so that for all future apps, `obscureText` is a bool

I don't know what `password` is needed as it seems to be the exact same as `obscureText` at least in this case.